apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1199,6 +1202,30 @@ type FIPSConfig struct {
 	CustomFIPSConfig *CustomConfig `json:"customFIPSConfig,omitempty"`
 }
 
+type AgentSidecarInjectionConfig struct {
+	// Enabled enables Sidecar injections.
+	// Default: true
+	// +optional
+	Enabled *bool `json:"enabled"`
+	// ClusterAgentCommunicationEnabled enables communication between Agent sidecars and the Cluster Agent.
+	// Default : true
+	// +optional
+	ClusterAgentCommunicationEnabled *bool `json:"clusterAgentCommunicationEnabled,omitempty"`
+	// Provider defines the provider for the Datadog Agent sidecar injection.
+	// Currently only "fargate" is supported.
+	// If not specified, It is automatically be set to "fargate".
```

Comment:
This is not accurate. Default is empty

https://github.com/DataDog/helm-charts/blob/5fdc9079d5645eaf995c7c8736cf2190798efc67/charts/datadog/values.yaml#L1117-L1119